contributing/STYLE.md

```diff
@@ -0,0 +1,171 @@
+      Cmdlet          New-FileCatalog                                    3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Protect-CmsMessage                                 3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Set-Acl                                            3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Set-AuthenticodeSignature                          3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Set-ExecutionPolicy                                3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Test-FileCatalog                                   3.0.0.0    Microsoft.PowerShell.Security
+      Cmdlet          Unprotect-CmsMessage                               3.0.0.0    Microsoft.PowerShell.Security
+      ```
+
+## Lists
+
+* Do not end list items with a period (unless they contain multiple sentences)
+* If your list contains multiple sentences, consider using a header 3/4/5 (as applicable) underneath your primary idea.
+
+If you want multiple lines within a single list element, format your list as follows:
+
+```markdown
+1. For the first element (like this one), insert a space after the 1.
+
+   To include a second element (like this one), insert a line break after the first and align indentations.
+   The indentation of the second element must line up with the first character after the numbered list marker.
+
+1. The next numbered item starts here.
+```
+
+to get this output:
+
+1. For the first element (like this one), insert a space after the 1.
+
+   To include a second element (like this one), insert a line break after the first and align indentations.
+   The indentation of the second element must line up with the first character after the numbered list marker.
+
+1. The next numbered item starts here.
+
+## Links
+
+* Avoid using bare URLs. Links should use MarkDown syntax `[friendlyname](url)`
+* Links should have a a friendly name when applicable, most likely the title of the linked page
+  * **Exception**: Links to non-Microsoft sites can be bare URLs
+* All items in the "related links" section at the bottom should be hyperlinked.
+
+## Using relative links
+
+You should use relative links when linking to other content that is hosted on **docs.microsoft.com**.
+When the content is in the same repo, the relative links are simple to calculate.
+Note that the path to cmdlet reference is created by our publishing system.
+There are special rules for linking to reference topics from conceptual topics.
+
+\[TO DO\] - document special rules
+
+<!-- External URLs -->
+[pascal-case]: https://en.wikipedia.org/wiki/PascalCase
+[issue1806]: https://github.com/PowerShell/PowerShell-Docs/issues/1806
+[atx]: https://github.github.com/gfm/#atx-headings
+[about-example]: https://github.com/PowerShell/PowerShell-Docs/blob/staging/reference/5.1/Microsoft.PowerShell.Core/About/about_Comparison_Operators.md
```

Comment:
technically this isn't an external URL

Reply:
@zjalexander - that is a comment that won't render. I created these reference links to shorten the link text within the flow of the paragraphs.